### ISSUE TITLE
java.lang.UnsupportedClassVersionError on JDK 8 on 0.9.0-M2

### ISSUE THREAD

**Issue by ngbinh** [scalapb/ScalaPB#568]
Here is the more complete error.
```
java.lang.UnsupportedClassVersionError: scalapb/options/compiler/Scalapb has been compiled by a more recent version of the Java Runtime (class file version 54.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

Because support for JDK 11 is incomplete for Scala 2.12.8, it would be great if Scalapb continue to support JDK 8. https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html

**Comment by thesamet** [comment]
This is unintentional, see #567. Please stay on 0.9.0-M1 for the time being. The only difference is a minor feature.

**Comment by ngbinh** [comment]
thanks!

**Comment by thesamet** [comment]
This is fixed in 0.9.0-M3.
